src/screens/Library/components/InstallModal/index.tsx

```diff
@@ -196,6 +205,7 @@ export default function InstallModal({
         )
       }
       const regexp = new RegExp(/[:|/|*|?|<|>|\\|&|{|}|%|$|@|`|!|™|+|']/, 'gi')
+      console.log(gameInfo, gameData)
```

Comment:
Remember to remove before mergin 👍🏽

Reply:
I missed that one. Maybe we should add eslint rules to ban console.log?

Reply:
yes, we should 👍🏽